query-engine/driver-adapters/src/wasm/to_js.rs

```diff
@@ -0,0 +1,27 @@
+use serde::Serialize;
+use serde_wasm_bindgen::Serializer;
+use wasm_bindgen::{JsError, JsValue};
+
+// `serialize_missing_as_null` is required to make sure that "empty" values (e.g., `None` and `()`)
+//  are serialized as `null` and not `undefined`.
+// This is due to certain drivers (e.g., LibSQL) not supporting `undefined` values.
+static DEFAULT_SERIALIZER: Serializer = Serializer::new().serialize_missing_as_null(true);
```

Comment:
Re: https://github.com/prisma/prisma-engines/pull/4649/files#r1453663258

In https://github.com/prisma/prisma-engines/pull/4648, I'll probably need to add `.serialize_large_number_types_as_bigints(true)` in here.

No biggie :)

Reply:
Do you want me to include it already before merging?